src/util/time.rs

```diff
@@ -43,6 +43,15 @@ pub fn duration_to_nanos(d: Duration) -> u64 {
     d.as_secs() * 1_000_000_000 + nanos
 }
 
+/// Get the current timestamp in seconds.
+#[inline]
+pub fn time_now_sec() -> u64 {
+    SystemTime::now()
```

Comment:
I worry about `SystemTime`s may vary in cluster if the ntp is not enabled.

Reply:
It doesn't need to compare with others, only compare himself. if the leader change the interval will not in a reasonable range, we will ignore it.

Reply:
maybe you can add another coarse time and use it to reduce the system call load.

Reply:
@siddontang in code https://github.com/rust-lang/rust/blob/1eab9c5a1b85db4ca850114814a7df3e289fcd67/src/libstd/sys/unix/time.rs#L353-L364.  it uses the `clock_gettime` function to get time, that exported by the vDSO.